File: 01 string manipulation/python/executer.py
import time

class Executer:
    def __init__(self, tests, function) -> None:
        self.tests = tests
        self.function = function

    def run(self):
        for test in self.tests:
            start = time.process_time_ns()
            result = self.function(test)
            end = time.process_time_ns()
            print(f"Input: {test}\nOutput: {result}\nTime: {(end - start)}ns\n")
        

File: 01 string manipulation/js/executer.js
const executer = (tests, callback) => {
    tests.forEach(test => {
        const start = window.performance.now();
        const result = callback(test);
        const end = window.performance.now();
        console.log(`Input: ${test}\nOutput:${result}\nTime: ${end - start}ns\n`);
    });
}